### PR TITLE
bust awc cache keys

### PR DESCRIPTION
**Pull request by calellowitz**
@Rohit25negi 
This is quick and dirty way to prevent zeros on the dashboard, but it busts the cache for all keys related to the  "CAS REACH" page of the program summary as soon as the aggregation ends.

### CODE REVIEW

**Comment by calellowitz** [comment]
cc: @emord

**Review by Rohit25negi — APPROVED** [review]
We might wanna do the proper thing for entire dashboard later. But this is good for first go. AWesome

**Review by kaapstorm — APPROVED** [review]

File: custom/icds_reports/tasks.py [hunk 1]
```diff
@@ -256,6 +256,8 @@ def move_ucr_data_into_aggregation_tables(date=None, intervals=2):
                             ).apply_async()
 
             res_awc.get()
+            reach_keys = cache.keys('*cas_reach_data*')
```
[the remark this answers is not in the document]
Comment (kaapstorm) [reply custom/icds_reports/tasks.py]: :thinking: Stickler seems wrong about that.

**Comment by emord** [comment]
This is inside the for loop for months. I think there's a chance that it fails after the first month's aggregation and before the second.

What do you think about having this in another task and calling it async. that way the agg script doesn't have another dependency on redis

**Comment by calellowitz** [comment]
@emord 
good point. fixed

File: custom/icds_reports/tasks.py [hunk 2]
```diff
@@ -970,3 +972,8 @@ def create_mbt_for_month(state_id, month):
             icds_file, _ = IcdsFile.objects.get_or_create(blob_id='{}-{}-{}'.format(helper.base_tablename, state_id, month), data_type='mbt_{}'.format(helper.base_tablename))
             icds_file.store_file_in_blobdb(f, expired=THREE_MONTHS)
             icds_file.save()
+
+@task(queue='icds_dashboard_reports_queue')
```
Comment (emord) [inline custom/icds_reports/tasks.py]: interesting choice of queue. was expecting the same agg queue. seems fine as long as there aren't any backups

**Review by emord — APPROVED** [review]

File: custom/icds_reports/tasks.py (hunk 2 above)
Comment (calellowitz) [reply custom/icds_reports/tasks.py]: oh, yeah. that was not the queue i intended, i copy pasted from the mbt part, which also appears to be in the wrong queue. although i sort of like the idea that both of those don't trade off with the aggregation.

**Pull request merged**